### PR TITLE
update decimal to 2.0

### PR DESCRIPTION
**Pull request by peaceful-james**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -36,10 +36,10 @@ defmodule Numbers.Mixfile do
     [
       {:coerce, "~> 1.0"},
 
-      {:decimal, "~> 1.9", optional: true},
+      {:decimal, "~> 2.0", optional: true},
 
       {:ex_doc, "~> 0.19", only: [:docs], runtime: false},
-      {:inch_ex, only: :docs}                  # Inch CI documentation quality test.
+      {:inch_ex, "~> 2.0.0", only: :docs}                  # Inch CI documentation quality test.
```
Comment (Qqwy) [inline mix.exs]: Is there a reason to restrict the version here? It probably should not be part of this PR.

File: mix.exs [hunk 2]
```diff
@@ -36,10 +36,10 @@ defmodule Numbers.Mixfile do
     [
       {:coerce, "~> 1.0"},
 
-      {:decimal, "~> 1.9", optional: true},
+      {:decimal, "~> 2.0", optional: true},
```
Comment (Qqwy) [inline mix.exs]: Could we change it to the more inclusive `~> 1.9 or ~> 2.0`? I'm pretty sure that for whatever reason there are packages which for other reasons are stuck on an older version of Decimal.

AFAICS none of the backwards-incompatible changes in Decimal 2.0 affect Numbers, at least :-).

**Review by Qqwy — CHANGES_REQUESTED** [review]

**Comment by peaceful-james** [comment]
@Qqwy I changed as requested. It should be good to merge/rebase now.

**Comment by Qqwy** [comment]
Version `5.2.4` which includes these changes has been published to Hex.

Thank you very much for your PR! :green_heart:
